tools/mo/requirements_onnx.txt

```diff
@@ -1,8 +1,4 @@
+-c ../../src/bindings/python/constraints.txt
+-r ./requirements.txt
```

Comment:
I think you need to create separate `requirements_extra.txt` file to move all overlappings among extra files.

Reply:
Currently all the overlapping requirements are contained in `requirements.txt`, which is linked here as `-r ./requirements.txt`. Running `pip install -r requirements_onnx.txt` will also install everything from the linked `requirements.txt`.

Reply:
They are anyway different because extra installation requires to have fixed upper-bound version, in the mean time. the default installation does not have it. So we cannot use `requirement.txt` file for collecting such overlapping items.

Reply:
This makes sense now. I reverted the changes.